README.md

```diff
@@ -200,12 +200,12 @@ For convenience aliases have been provided for all supported request methods.
 ###### NOTE
 When using the alias methods `url`, `method`, and `data` properties don't need to be specified in config.
 
-### Concurrency
+~~Concurrency~~
```

Comment:
Better to add a sentence here to explain why they are removed. Say,

```
Deprecated. Use `Promise.all` to replace them.
```

Reply:
Mended